packages/parser-jsx/src/parser.ts

```diff
@@ -206,6 +206,52 @@ const addChild = (data: ElementData | TextData, parent: JSXElement, children: Ch
     children.set(parent, list);
 };
 
+/**
+ * Is the node a list container (`<ul>/<ol>`).
+ */
+const isListNode = (node: JSXElement | JSXAttribute | undefined): node is JSXElement => {
```

Comment:
Minor nit, not blocking.
```suggestion
const isListNode = (node?: JSXElement | JSXAttribute): node is JSXElement => {
```

Reply:
Thanks, done!